e2e/txserver/zeta_tx_server.go

```diff
@@ -557,6 +579,33 @@ func (zts *ZetaTxServer) SetAuthorityClient(authorityClient authoritytypes.Query
 	zts.authorityClient = authorityClient
 }
 
+// initializeLiquidityCap initializes the liquidity cap for the given coin with a large value
+func (zts ZetaTxServer) initializeLiquidityCap(zrc20 string) error {
+	liquidityCap := sdktypes.NewUint(1e18).MulUint64(1e12)
```

Comment:
nit: having a `const.go` file per package would group up all the "magic numbers" into one single file, making it more readable and upgradable.

Reply:
Yeah I agree although for this one, this is mostly just about setting a number high enough, no real importance on the number itself